components/net/http_cache.rs

```diff
@@ -85,6 +103,17 @@ struct CachedMetadata {
     pub status: Option<(u16, Vec<u8>)>
 }
 
+impl MallocSizeOf for CachedMetadata {
```

Comment:
I've filed https://github.com/servo/servo/issues/20401 about making it easier to deal with structures like this, but it would be nice to use the same `Measurable` pattern as CachedResource here.